src/basic/slow-code.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#pragma once
+
+#include "log.h"
+#include "string-util.h"
+#include "time-util.h"
+
+#include "macro.h"
+
+typedef struct slow_code_detector {
+        uint32_t timeout_sec;
+        uint32_t count;
+        usec_t current_time;
+        size_t line_num;
```

Comment:
we usually used `unsigned` type wherever `__LINE__` is used

Reply:
updated.